docs/videos/healthcare-and-life-sciences/Factor-Xa-example/submit.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#SBATCH -N 2
+#SBATCH --ntasks-per-node 30
+#SBATCH --partition compute
+
+PDB_FILE=1FJS.pdb
+PROTEIN="${PDB_FILE%.*}"
+
+echo $PDB_FILE
+echo $PROTEIN
+
+# Activate GROMACS environment
+source /apps/spack/share/spack/setup-env.sh
+spack env activate gromacs
+
+# Check that gmx_mpi exists
+which gmx_mpi
+
+# Create hostname file
+scontrol show hostnames "${SLURM_JOB_NODELIST}" >hostfile
+
+# Prepare Inputs
+grep -v -e HETATM -e CONECT ${PDB_FILE} >${PROTEIN}_protein.pdb
+
+# Generate Topology
+gmx_mpi pdb2gmx -f ${PROTEIN}_protein.pdb -o ${PROTEIN}_processed.gro -water tip3p -ff "charmm27"
+
+# Solvate System
+gmx_mpi editconf -f ${PROTEIN}_processed.gro -o ${PROTEIN}_newbox.gro -c -d 1.0 -bt dodecahedron
+gmx_mpi solvate -cp ${PROTEIN}_newbox.gro -cs spc216.gro -o ${PROTEIN}_solv.gro -p topol.top
+
+# Add ions
+gmx_mpi grompp -f config/ions.mdp -c ${PROTEIN}_solv.gro -p topol.top -o ions.tpr
+printf "SOL\n" | gmx_mpi genion -s ions.tpr -o ${PROTEIN}_solv_ions.gro -conc 0.15 -p topol.top -pname NA -nname CL -neutral
+
+# Launch MPI jobs
+
+# Run energy minimization
+gmx_mpi grompp -f config/emin-charmm.mdp -c ${PROTEIN}_solv_ions.gro -p topol.top -o em.tpr
+mpirun -n 60 -hostfile hostfile -ppn 30 gmx_mpi mdrun -v -deffnm em
```

Comment:
if you are running this with Slurm, why do you need the hostfile? (see lines 33, 34, 58, 62, 66). Also, why do you need ppn if the nodes already have hyperthreading off?  (also, see line 17)
Specifying these may reduce make this script less future proof. Say you wanted to run this in C3, you would maybe have a different setup.

Reply:
I agree with your point and if you want me to make the change I will, but my preference would be to leave this as the next pr (#891 already open) will be modifying all of this to run with gpus and openmpi. Since I know the current state works, I would rather leave it as is and capture this working step before the next move to GPUs. If I update to remove hostname and ppn I would want to rerun tests.

Reply:
ack